runelite-client/src/main/java/net/runelite/client/plugins/roguesden/RoguesDenOverlay.java

```diff
@@ -24,17 +24,18 @@
  */
 package net.runelite.client.plugins.roguesden;
 
-import java.awt.Color;
-import java.awt.Dimension;
-import java.awt.Graphics2D;
-import java.awt.Polygon;
+import java.awt.*;
```

Comment:
You should not use wildcard imports. Make sure you only import what you are using.